core/src/main/scala/org/apache/spark/sql/TiExtensions.scala

```diff
@@ -110,4 +112,16 @@ object TiExtensions {
     tiExtensions
   }
 
+  /**
+   * check that telemetry is on
+   * default on
+   *
+   * @param sparkSession
+   * @return
+   */
+  def telemetryEnable(sparkSession: SparkSession): Boolean = {
+    sparkSession.sparkContext.conf
+      .get("spark.tispark.telemetry.enable", "true")
```

Comment:
I think it is better to define a const whose value is `spark.tispark.telemetry.enable` rather than use it directly

Reply:
yep. I have used a const `TELEMETRY_ENABEL` to replace it